pontoon/base/models/entity.py

```diff
@@ -888,36 +887,43 @@ def for_project_locale(
             )
 
             # Search in source strings
-            if not search_translations_only:
-                entity_filters = (
-                    Q(
-                        Q(resource__format="ftl")
-                        & (Q(**{f"string__{i}regex": rf"{r}{y}{escape(s)}{y}{o}"}))
-                    )
-                    | Q(
-                        ~Q(resource__format="ftl")
-                        & (
-                            Q(**{f"string__{i}regex": rf"{y}{escape(s)}{y}"})
-                            | Q(**{f"string_plural__{i}regex": rf"{y}{escape(s)}{y}"})
+            entity_filters = (
+                (
+                    Q(pk=None)  # Ensures that no source strings are returned
```

Comment:
Why is this not simply Q(), like in line 878?

Reply:
During testing, it seemed that using Q() returns _all_ objects.

I found this [StackOverflow question](https://stackoverflow.com/questions/29900386/how-to-construct-django-q-object-matching-none) which was about the same problem I was encountering, and switching to Q(pk=None) now correctly returns no matches.

Reply:
Oh. Oh! 🤦